autoload/mucomplete/auto.vim

```diff
@@ -5,13 +5,36 @@
 let s:save_cpo = &cpo
 set cpo&vim
 
+fun! mucomplete#auto#start_timer() abort
+  if exists('s:completion_timer')
+    call mucomplete#auto#stop_timer()
+  endif
+
+  let l:delay = max([20, get(g:, 'mucomplete#delayed_completion', 0)])
+  let s:completion_timer = timer_start(l:delay, {-> mucomplete#auto#auto_complete() })
```

Comment:
Works fine in my neovim. I'm using latest unstable tho... I would have to create a wrapping function because it provides timer property.

Reply:
I switched to function name. Also had to add optional param (`...`) to the `mucomplete#auto#auto_complete` because timer_start passes down timer property.